pkg/windows/windows.go

```diff
@@ -401,13 +399,6 @@ func (vm *windows) Reinitialize() error {
 }
 
 func (vm *windows) EnsureRequiredServicesStopped() error {
-	// TODO: In this transitional period of migrating services to WICD's responsibility, WICD must be stopped first to
-	//       ensure it does not start any of these stopped services. This code should be removed as part of
-	//       https://issues.redhat.com/browse/WINC-733
-	svc := &service{name: wicdServiceName}
-	if err := vm.ensureServiceNotRunning(svc); err != nil {
-		return errors.Wrapf(err, "could not stop service %s", wicdServiceName)
-	}
 	for _, svcName := range append(RequiredServices, AzureCloudNodeManagerServiceName) {
```

Comment:
WICD should be stopping these services, not WMCO.

Reply:
Shouldnt this whole method should be removed from the Windows interface?

Reply:
@sebsoto This method is now only being used in the WICD [bootstrap call ](https://github.com/openshift/windows-machine-config-operator/blob/be61290e3f49cf006b95f003b48a1e559eb348cf/pkg/windows/windows.go#L469) and configure not during the cleanup process. Are you saying this needs to be replaced?

Reply:
WMCO should not be stopping any services besides the WICD controller service. You should invoke WICD cleanup if the services started by WICD need to be stopped.